### PR TITLE
publiccloud: Boottime thresholds per flavor

### PR DESCRIPTION
**Pull request by cfconrad**
We get very different timings for boottime validation. So we need to
specify thresholds per FLAVOR.

- Related ticket: https://progress.opensuse.org/issues/56462
- Verification run: TBD

### CODE REVIEW

**Comment by cfconrad** [comment]
Also fixed http://openqa-perf.qa.suse.de/d/A-z8Cz0Zz/systemd-analyze-blame view.

File: tests/publiccloud/boottime.pm [hunk 1]
```diff
@@ -119,6 +130,18 @@ sub run {
         ssh_access_hard => 60,
     };
 
+    my $threshold_by_flavor = {
+        qr/^(Azure-Standard|Azure-Basic)$/ => {
+            overall   => 3 * 60,
+            userspace => 3 * 60,
```
Comment (jlausuch) [inline tests/publiccloud/boottime.pm:136]: overall should be higher than userspace, right? 
Why are you setting now overall to 180 if it was 160 in the default list?

**Comment by jlausuch** [comment]
According to the discussion with PC team, the idea was to have it per provider, not per flavor. I know doing it per flavor is more flexible, but I wonder if it would be clearer to have only 3 values for each parameter (1 value per provider), something like a matrix, instead of patching some default values.

**Comment by asmorodskyi** [comment]
> According to the discussion with PC team, the idea was to have it per provider, not per flavor. I know doing it per flavor is more flexible, but I wonder if it would be clearer to have only 3 values for each parameter (1 value per provider), something like a matrix, instead of patching some default values.

I second that , instead of programmatically nice looking thingy I would prefer to have clear matrix split by provider

**Comment by cfconrad** [comment]
I don't like the idea of having it per provider. At least we should separate between BYOS and On-Demand. Look at Azure http://s.qa.suse.de/KS1y there is a diff of 1min between them.

**Comment by cfconrad** [comment]
> According to the discussion with PC team, the idea was to have it per provider, not per flavor.

They also had the idea to separate per instance-type. So I guess this isn't the last iteration and I would keep the config-matrix as small as possible.

File: tests/publiccloud/boottime.pm (hunk 1 above)
Comment (cfconrad) [reply tests/publiccloud/boottime.pm:136]: ok

**Pull request merged**